### DISCUSSION TITLE
getting_started/install

### DISCUSSION THREAD

**Discussion by giscus** [Air-duino/document#9] [comment]
# getting_started/install

此文档旨在介绍关于 AirMCU 的相关安装方式。 Arduino IDE 的安装 Arduino IDE支持Windows、Mac OS X及Linux等平台，下载地址为Software > Arduino (https://www.arduino.cc/en/software)，本文仅介绍Windows下的安装方式，其他系统大同小异，请读者自行研究...

https://arduino.luatos.com/getting_started/install.html

**Comment by xuyajian198787** [comment]
FQBN: AirM2M:AirMCU:Air401Dev
使用平台的 ‘Air401Dev’ 开发板，在列出的文件夹中：C:\Users\New Farmer\AppData\Local\Arduino15\packages\AirM2M\hardware\AirMCU\0.6.2
使用平台的 ‘AirMCU’ 代码，在列出的文件夹中：C:\Users\New Farmer\AppData\Local\Arduino15\packages\AirM2M\hardware\AirMCU\0.6.2

"C:\\Users\\New Farmer\\AppData\\Local\\Arduino15\\packages\\AirM2M\\hardware\\AirMCU\\0.6.2\\system\\extras\\prebuild.bat" "C:\\Users\\New Farmer\\AppData\\Local\\Temp\\arduino\\sketches\\AA665A045BE9651D895BBB715698FA4A" "C:\\Users\\New Farmer\\AppData\\Local\\Temp\\.arduinoIDE-unsaved20231115-6432-1nxyxl4.fl8e\\sketch_dec15a" "C:\\Users\\New Farmer\\AppData\\Local\\Arduino15\\packages\\AirM2M\\hardware\\AirMCU\\0.6.2"
'C:\Users\New' �����ڲ����ⲿ���Ҳ���ǿ����еĳ���
���������ļ���

exit status 1

Compilation error: exit status 1
请问这个编译失败怎么回事

**Reply by HalfSweet** [reply]
https://arduino.luatos.com/FAQ/chinese_path.html

**Comment by conglinyizhi** [comment]
平台安装失败：'Air MCU:0.6.4' 2 UNKNOWN: Get "https://arduino.luatos.com/AirISP-0.2.0-linux-x86_64.tar.gz": tls: failed to verify certificate: x509: certificate signed by unknown authority

怪了，浏览器访问没有问题的

**Reply by SunZongming** [reply]
我也遇到类似的错误了，请问解决了吗？
"https://arduino.luatos.com/package_air_cn_index.json": tls: failed to verify certificate: x509: certificate has expired or is not yet valid: : https://arduino.luatos.com/package_air_cn_index.json

**Reply by conglinyizhi** [reply]
> 我也遇到类似的错误了，请问解决了吗？ "https://arduino.luatos.com/package_air_cn_index.json": tls: failed to verify certificate: x509: certificate has expired or is not yet valid: : https://arduino.luatos.com/package_air_cn_index.json

并没有，我已经更换了方案

**Comment by daweipeng8** [comment]
开发板安装失败。怎么回事
下载 https://arduino.luatos.com/package_air_cn_index.json 时出错
 下载 https://arduino.luatos.com/xpack-arm-none-eabi-gcc-13.2.1-1.1-win32-x64.zip 时出错
